{"name": "test_registration_flow", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'Dynamic' has no attribute 'environment'", "trace": "@pytest.fixture(autouse=True)\n    def add_allure_env():\n>       allure.dynamic.environment(host=\"pizzeria.skillbox.cc\")\nE       AttributeError: type object 'Dynamic' has no attribute 'environment'\n\npizzeria_tests\\conftest.py:12: AttributeError"}, "start": 1744229794780, "stop": 1744229794780, "uuid": "3b0153ca-8d1d-4cb5-b3d6-6fbde8c066e4", "historyId": "7c80ca35f9890c170998039a6b6ec2d4", "testCaseId": "7c80ca35f9890c170998039a6b6ec2d4", "fullName": "pizzeria_tests.tests.test_main_flow#test_registration_flow", "labels": [{"name": "parentSuite", "value": "pizzeria_tests.tests"}, {"name": "suite", "value": "test_main_flow"}, {"name": "host", "value": "Ittekimasu"}, {"name": "thread", "value": "3628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pizzeria_tests.tests.test_main_flow"}]}